app/presenters/person_presenter.rb

```diff
@@ -30,10 +30,37 @@ def has_policy_responsibilities?
   end
 
   def announcements
-    announcements =
-      model.published_speeches.with_translations(I18n.locale).limit(10).map { |s| SpeechPresenter.new(s, context) } +
-      model.published_news_articles.with_translations(I18n.locale).limit(10).map { |na| NewsArticlePresenter.new(na, context) }
-    announcements.sort_by { |a| a.public_timestamp.to_datetime }.reverse[0..9]
+    search_results = Whitehall.search_client.search(
+      filter_people: model.slug,
+      count: 10,
+      order: "-public_timestamp",
+      reject_content_purpose_supergroup: "other",
+      fields: %w[title link content_store_document_type public_timestamp]
+    )["results"]
+
+    documents = []
+
+    search_results.each do |item|
```

Comment:
Could do a map here :-).

Reply:
Yeah. I totes copy-pasted this from collections.